ui/admin-portal/src/app/services/auth.service.ts

```diff
@@ -149,11 +149,30 @@ export class AuthService {
   }
 
   isCandidateOurs(candidate: ShortCandidate): boolean {
-    return true; //todo
+    let ours = false;
+    const loggedInUser = this.getLoggedInUser()
```

Comment:
```suggestion
    const loggedInUser: User = this.getLoggedInUser()
```

Reply:
Same comment as above - really no need for explicit type